presto-main/src/main/java/com/facebook/presto/execution/buffer/OutputBufferMemoryManager.java

```diff
@@ -95,7 +95,12 @@ public double getUtilization()
         return bufferedBytes.get() / (double) maxBufferedBytes;
     }
 
-    public boolean isFull()
+    public synchronized boolean isOverutilized()
+    {
+        return isBufferFull();
+    }
+
+    private synchronized boolean isBufferFull()
```

Comment:
I created `isBufferFull` for internal use, which will help me with what I am working on right now. For now what matters is to make `isOverutilized` synchronized.